src/Fantomas.Tests/NumberOfItemsTests.fs

```diff
@@ -0,0 +1,268 @@
+              ArrayOrListMultilineFormatter = NumberOfItems
+              MultilineBlockBracketsOnSameColumn = true }
+    |> prepend newline
+    |> should equal """
+let xs =
+    [
+        a
+        b
+        c
+    ]
+
+let ys =
+    [
+        AReallyLongExpressionThatIsMuchLongerThan50Characters
+    ]
+
+f
+    xs
+    [
+        x
+        y
+        z
+    ]
+
+List.map (fun x -> x * x)
+    [
+        1
+        2
+    ]
+"""
+
+[<Test>]
+let ``number of items sized arrays with block brackets on same column are formatted properly`` () =
+    formatSourceString false """
+let xs = [| a; b; c |]
+let ys = [| AReallyLongExpressionThatIsMuchLongerThan50Characters |]
+f xs [| x; y; z |]
+List.map (fun x -> x * x) [|1;2;|]
+    """
+        { config with
+              ArrayOrListMultilineFormatter = NumberOfItems
+              MultilineBlockBracketsOnSameColumn = true }
+    |> prepend newline
+    |> should equal """
+let xs =
+    [|
+        a
+        b
+        c
+    |]
+
+let ys =
+    [|
+        AReallyLongExpressionThatIsMuchLongerThan50Characters
+    |]
+
+f
+    xs
+    [|
+        x
+        y
+        z
+    |]
+
+List.map (fun x -> x * x)
+    [|
+        1
+        2
+    |]
+"""
+
+[<Test>]
+let ``number of items sized Elmish lists with block brackets on same column are formatted properly`` () =
+    formatSourceString false """
+f [ a; b; c ] [ x; y; z ]
+g [ longValueThatIsALotOfCharactersSoooooLong ] [ longValueThatIsALotOfCharactersSoooooLong ]
+h [ longValueThatIsALotOfCharactersSoooooLong; longValueThatIsALotOfCharactersSoooooLong ] [ longValueThatIsALotOfCharactersSoooooLong ]
+    """
+        { config with
+              ArrayOrListMultilineFormatter = NumberOfItems
+              MultilineBlockBracketsOnSameColumn = true }
+    |> prepend newline
+    |> should equal """
+f [
+    a
+    b
+    c
+  ] [
+    x
+    y
+    z
+]
+
+g [
+    longValueThatIsALotOfCharactersSoooooLong
+  ] [
+    longValueThatIsALotOfCharactersSoooooLong
+]
+
+h [
+    longValueThatIsALotOfCharactersSoooooLong
+    longValueThatIsALotOfCharactersSoooooLong
+  ] [
+    longValueThatIsALotOfCharactersSoooooLong
+]
+"""
+
+let ``number of items sized Elmish lists with single argument web mode are formatted properly`` () =
+    formatSourceString false """
+f [ a; b; c ]
+g [ longValueThatIsALotOfCharactersSoooooLong ]
+h [ longValueThatIsALotOfCharactersSoooooLong; longValueThatIsALotOfCharactersSoooooLong ]
+    """
+        { config with
+              ArrayOrListMultilineFormatter = NumberOfItems
+              SingleArgumentWebMode = true }
+    |> prepend newline
+    |> should equal """
+f [
+    a
+    b
+    c
+]
+
+g [
+    longValueThatIsALotOfCharactersSoooooLong
+]
+
+h [
+    longValueThatIsALotOfCharactersSoooooLong
+    longValueThatIsALotOfCharactersSoooooLong
+]
+"""
+
+let ``number of items sized Elmish lists with single argument web mode and multiline block brackets on same column are formatted properly`` () =
```

Comment:
Add the `[<Test>]` attribute please, I was already very puzzled why this could work 😅.
The test is not running, so that is why...

Reply:
Added.